AST.fs

```diff
@@ -153,6 +168,22 @@ module Pretty =
     open Starling.Core.Model.Pretty
     open Starling.Core.Var.Pretty
 
+    /// Some ANSI color codes for nice outputting during debugging
+    /// wraps x as "<[line n: x]>" with color formats
+    let pprint_lines = false
```

Comment:
Also, it might be worth making colouring a primitive in the pretty-printer, that'd make it easier to turn colour off for things that don't support it.